addons/web_editor/static/src/js/editor/odoo-editor/test/spec/copyPaste.test.js

```diff
@@ -69,6 +69,70 @@ describe('Copy', () => {
+                contentBefore: '<p><strong><em><u><font class="text-o-color-1">hello [there]</font></u></em></strong></p>',
+                stepFunction: async editor => {
+                    const clipboardData = new DataTransfer();
+                    triggerEvent(editor.editable, 'copy', { clipboardData });
+                    window.chai.expect(clipboardData.getData('text/plain')).to.be.equal('there');
+                    window.chai.expect(clipboardData.getData('text/html')).to.be.equal('<p><strong><em><u><font class="text-o-color-1">there</font></u></em></strong></p>');
+                    window.chai.expect(clipboardData.getData('text/odoo-editor')).to.be.equal('<p><strong><em><u><font class="text-o-color-1">there</font></u></em></strong></p>');
+                },
+            });
+        });
+        it('should copy a list selection as text/plain, text/html and text/odoo-editor', async () => {
+            await testEditor(BasicEditor, {
+                contentBefore: '<ul><li>[First]</li><li>Second</li>',
+                stepFunction: async editor => {
+                    const clipboardData = new DataTransfer();
+                    triggerEvent(editor.editable, 'copy', { clipboardData });
+                    window.chai.expect(clipboardData.getData('text/plain')).to.be.equal('First');
+                    window.chai.expect(clipboardData.getData('text/html')).to.be.equal('<li>First</li>');
+                    window.chai.expect(clipboardData.getData('text/odoo-editor')).to.be.equal('<li>First</li>');
+                },
+            });
+            await testEditor(BasicEditor, {
+                contentBefore: '<ul><li>First [List]</li><li>Second</li>',
+                stepFunction: async editor => {
+                    const clipboardData = new DataTransfer();
+                    triggerEvent(editor.editable, 'copy', { clipboardData });
+                    window.chai.expect(clipboardData.getData('text/plain')).to.be.equal('List');
+                    window.chai.expect(clipboardData.getData('text/html')).to.be.equal('<li>List</li>');
+                    window.chai.expect(clipboardData.getData('text/odoo-editor')).to.be.equal('<li>List</li>');
+                },
+            });
+            await testEditor(BasicEditor, {
+                contentBefore: '<ul><li>[First</li><li>Second]</li>',
+                stepFunction: async editor => {
+                    const clipboardData = new DataTransfer();
+                    triggerEvent(editor.editable, 'copy', { clipboardData });
+                    window.chai.expect(clipboardData.getData('text/plain')).to.be.equal('First\nSecond');
+                    window.chai.expect(clipboardData.getData('text/html')).to.be.equal('<ul><li>First</li><li>Second</li></ul>');
+                    window.chai.expect(clipboardData.getData('text/odoo-editor')).to.be.equal('<ul><li>First</li><li>Second</li></ul>');
+                },
+            });
```

Comment:
I suggest you move the test case to a dedicated "it" block, with something like this as text: "should copy the selection as a list with two list items".

Reply:
@cammarosano updated the requested changes!

Reply:
Nice! Thank you!